src/containers/notification/store/sagas.ts

```diff
@@ -536,8 +536,25 @@ function* watchTogglePanel() {
   })
 }
 
+// On Native App open, clear the notification badges
+function* notificationsInit() {
+  try {
+    yield call(waitForBackendSetup)
+
+    const hasAccount = yield select(getHasAccount)
+    if (hasAccount && NATIVE_MOBILE) {
+      const message = new ResetNotificationsBadgeCount()
+      message.send()
+      yield call(AudiusBackend.clearNotificationBadges)
+    }
+  } catch (error) {
+    console.error(error)
+  }
+}
+
 export default function sagas() {
   return [
+    notificationsInit,
```

Comment:
nit, i would just exclude the saga entirely if not native mobile

const sagas = [ .... ]
if (NATIVE_MOBILE) sagas.push(resetNotificationBadgeCount)  << and rename to something explicit like

return sagas